Dockerfile

```diff
@@ -49,28 +49,21 @@ RUN apt-get update && apt-get install --no-install-recommends -y --force-yes \
   imagemagick \
   x11-apps \
   unzip \
-  openssh-client
+  openssh-client \
+  graphviz \
+  doxygen
+
+# Install java
+RUN mkdir -p /usr/share/man/man1
+RUN apt-get install --no-install-recommends -y --force-yes default-jre-headless
+
+# Install plantuml
+RUN apt-get install --no-install-recommends -y --force-yes plantuml
 
 # Add maximum backwards compatibility with MySQL 5.6
 RUN echo "[mysqld]" >> /etc/mysql/conf.d/z-pipelines-config.cnf && \
   echo 'sql_mode = "NO_ENGINE_SUBSTITUTION"' >> /etc/mysql/conf.d/z-pipelines-config.cnf
 
-# Install the Oracle client
```

Comment:
Why are you removing the oracle client?

Reply:
We use a (fake) MySQL database nowadays instead of Oracle during a build.